.github/workflows/get-installations.yaml

```diff
@@ -39,10 +42,20 @@ jobs:
       email: ${{ steps.repos.outputs.email }}
     steps:
       - id: new-repos
+        name: "Evaluate inputs"
         run: |
+          echo '::debug::repo: ${{ inputs.repo }}'
+          echo '::debug::newbies: ${{ inputs.newbies }}'
           {
             echo "new<<EOF"
-            echo '${{ inputs.newbies }}'
+            if [[ $(grep -c '/' <<< '${{ inputs.repo }}') -gt 0 ]]; then
+              echo '${{ inputs.repo }}' \
+              | tr ' ' '\n' \
+              | jq -R \
+              | jq -sc 'map(split("/") | {owner:.[0], name:.[1]})'
+            else
```

Comment:
`inputs.newbies` is expected to be a JSON list of objects like this:

```
[{"owner": "a", "repo": "arepo"}, {"owner": "b", "repo": "brepo"}]
```

This addition allows someone to specify `a/arepo b/brepo` in the workflow call to create the same json list.